debezium-ddl-parser/src/main/antlr4/io/debezium/ddl/parser/oracle/generated/PlSqlParser.g4

```diff
@@ -3576,7 +3576,8 @@ lock_table_element
     ;
 
 lock_mode
-    : ROW SHARE
+    : ACCESS SHARE
+    | ROW SHARE
```

Comment:
I don't think we need to change this as this DDL grammar is specific only to the Oracle connector and your change is something specific to PostgreSQL.

Reply:
@Naros didn't see that.  Will remove.